File: materias.js
var fs=require('fs');
var data=fs.readFileSync('materias.json', 'utf8');
var words=JSON.parse(data);
var materias = words.filter((x) =>  x.situacao === 'A cursar' & x.carga_horaria === 60 );

materias.forEach(f => {
    console.log(f.nome_disciplina);
});

File: materias.py
import json
import pandas as pd


with open("materias.json", encoding='utf-8') as meu_json:
    dados = json.load(meu_json)
    lst = list(dados)
   
materias = [n for n in lst if  n['situacao'] == "A cursar"] 
df = pd.DataFrame(materias)

df.to_csv("materias a cursar.csv")
for i in materias: print(i) 